docs/guide/advanced/sfc.md

```diff
@@ -295,6 +301,90 @@ The `i18n` custom blocks below of `json5` format:
 </i18n>
 ```
 
+### Define global format
+
+If you are using one of `@intlify/vite-plugin-vue-i18n`, `@intlify/vue-i18n-loader`  or `@intlify/rollup-plugin-vue-i18n`
+plugin on your project, you can also define the `lang` for all your inlined `i18n` custom blocks
+on all your `SFC` using the `defaultSFCLocale` option.
+
+On inlined `i18n` custom blocks that have specified the `lang` attribute, the `defaultSFCLang` is not applied.
+
+For example, in order to configure `yml` format on all your inlined `i18n` custom blocks on all your `SFC`:
+
+```html
+<!-- custom block equivalent to &lt;i18n lang="yml"> -->
+<i18n>
+en:
+  hello: Hello
+es:
+  hello: Hola
+</i18n>
+```
+
+just configure `defaultSFCLocale: "yml"` on your plugin configuration:
+
+#### Use global format with vite plugin
+
+```javascript
+import vue from '@vitejs/plugin-vue'
+import vueI18n from '@intlify/vite-plugin-vue-i18n'
+
+plugins: [
+  vue(),
+  vueI18n({
+    defaultSFCLocale: "yml",
+    /* other options */
+  })
+]
+```
+
+#### Use global format with webpack plugin
+
+```javascript
+module.exports = {
+  // ...
+  module: {
+    rules: [
+      {
+        test: /\.vue$/,
+        loader: 'vue-loader',
+      },
+      // for i18n resources (json/json5/yaml)
+      {
+        test: /\.(json5?|ya?ml)$/, // target json, json5, yaml and yml files
+        type: 'javascript/auto',
+        defaultSFCLocale: 'yml',
```

Comment:
In webpack, We don't still develop `defaultSSFCLang`.
Currently, we have a feature request.
https://github.com/intlify/bundle-tools/issues/24